streams/test-utils/src/main/java/org/apache/kafka/streams/TopologyTestDriver.java

```diff
@@ -333,22 +333,28 @@ public void onRestoreEnd(final TopicPartition topicPartition, final String store
             offsetsByTopicPartition.put(tp, new AtomicLong());
         }
         consumer.assign(partitionsByTopic.values());
+        final Map<TopicPartition, Long> startOffsets = new HashMap<>();
+        for (final TopicPartition topicPartition : partitionsByTopic.values()) {
+            startOffsets.put(topicPartition, 0L);
+        }
+        consumer.updateBeginningOffsets(startOffsets);
```

Comment:
Because we call `consumer.position()`  now, we need to fix the mock consumer setup in the TTD